### PR TITLE
Feat #33/keycloak access via http

### PR DESCRIPTION
**Pull request by mvhysko**

### CODE REVIEW

File: Dockerfile [hunk 1]
```diff
@@ -7,7 +7,7 @@ COPY config ./config
 COPY tsconfig.build.json ./
 COPY tsconfig.json ./
 COPY nest-cli.json ./
-COPY .env ./
+COPY ./examples/.env.example ./.env
```
Comment (derevnjuk) [inline Dockerfile:10]: Why did you revert these changes?

File: public/src/pages/auth/Login/Login.tsx [hunk 2]
```diff
@@ -152,10 +152,13 @@ export const Login: FC = () => {
...
               <p>
-                <b>clientId:</b> {oidcClient.clientId}
+                <b>client_id:</b> {oidcClient.clientId}
               </p>
               <p>
-                <b>clientSecret:</b> {oidcClient.clientSecret}
+                <b>client_secret:</b> {oidcClient.clientSecret}
+              </p>
+              <p>
+                <b>oidc_config/metadata URL:</b> {oidcClient.metadataUrl}
```
Comment (derevnjuk) [inline public/src/pages/auth/Login/Login.tsx:161]:
```suggestion
                <b>Openid-configuration URL:</b> {oidcClient.metadataUrl}
```

**Review by derevnjuk — APPROVED** [review]

File: Dockerfile (hunk 1 above)
Comment (mvhysko) [reply Dockerfile:10]: Oh, thank you!

File: public/src/pages/auth/Login/Login.tsx [hunk 3]
```diff
@@ -152,10 +152,13 @@ export const Login: FC = () => {
...
               <p>
-                <b>clientId:</b> {oidcClient.clientId}
+                <b>client_id:</b> {oidcClient.clientId}
               </p>
               <p>
-                <b>clientSecret:</b> {oidcClient.clientSecret}
+                <b>client_secret:</b> {oidcClient.clientSecret}
+              </p>
+              <p>
+                <b>oidc-config URL:</b> {oidcClient.metadataUrl}
```
Comment (derevnjuk) [inline public/src/pages/auth/Login/Login.tsx:161]: Don't use abbr here. Please keep the same wording
```suggestion
                <b>Openid-configuration URL:</b> {oidcClient.metadataUrl}
```

**Review by derevnjuk — APPROVED** [review]
